packages/storage-plus/README.md

```diff
@@ -202,15 +202,14 @@ A `Map` key can be anything that implements the `PrimaryKey` trait. There are a
  - `impl<'a, T: Endian + Clone> PrimaryKey<'a> for IntKey<T>`
 
 That means that byte and string slices, byte vectors, and strings, can be conveniently used as keys.
-Moreover, some other types can be used as well, like addresses and addresses references, pairs and triples, and
+Moreover, some other types can be used as well, like addresses and address references, pairs and triples, and
 integer types.
 
-If the key represents and address, we suggest using `&Addr` for keys in storage, instead of `String` or string slices. This implies doing address validation
-through `addr_validate` on any address passed in via a message, to ensure it's a legitimate address, and not random text
-which will fail later.
-
-Thus, `pub fn addr_validate(&self, &str) -> Addr` in `deps.api` can be used for address validation, and the returned
-`Addr` can be conveniently used as key in a `Map` or similar structure.
+If the key represents and address, we suggest using `&Addr` for keys in storage, instead of `String` or string slices.
```

Comment:
> If the key represents and address, we suggest ...
Hmm, this part confuses me.
Do you mean "represents and address [something, thing to address]"?

Reply:
It's "_an_ address". Will fix it.